src/content/Angular-CSharp/ClientApp/proxy.conf.js

```diff
@@ -13,6 +13,7 @@ const PROXY_CONFIG = [
       "/Identity",
       "/connect",
       "/ApplyDatabaseMigrations",
+      "/_framework/aspnetcore-browser-refresh.js"
```

Comment:
Do you need to list other paths that gets used here: https://github.com/dotnet/sdk/blob/main/src/BuiltInTools/BrowserRefresh/ApplicationPaths.cs

Reply:
Updated